server/server.go

```diff
@@ -81,7 +87,7 @@ func Serve(ctx context.Context, args []string) {
 		w.WriteHeader(http.StatusOK)
 		fmt.Fprintln(w, "OK")
 		// fmt.Printf("Data: %+v\n", bd)
-		logger.Info("API Data fetched", "data", bd, "provider", "meteo-blue")
+		logger.Debug("API Data fetched", "data", bd, "provider", "meteo-blue")
```

Comment:
Shouldn't we also write to response like in line 54

Reply:
Didn't change it because MB hasn't been properly implemented yet; it's merely a play area.

Reply:
Cool